src/Tools/AI Test Toolkit/src/SampleTests/DemoSuite/SentenceValidator.Codeunit.al

```diff
@@ -0,0 +1,35 @@
+namespace System.TestTools.AITestToolkit;
+codeunit 149031 SentenceValidator
+{
+    Access = Internal;
+    Subtype = Test;
+
+    [Test]
+    procedure AssertSentenceLengthFunction()
+    var
+        AITContext: Codeunit "AIT Test Context";
+        UserInput: Text;
+    begin
+        // AITContext.StartScenario('Scene1');
```

Comment:
Remove comments?

Reply:
The sample tests will not be shipped, they are here to help to understand the code better.